python/ray/_raylet.pyx

```diff
@@ -352,8 +352,12 @@ cdef class RayletClient:
 
     def prepare_actor_checkpoint(self, ActorID actor_id):
         cdef CActorCheckpointID checkpoint_id
-        check_status(self.client.get().PrepareActorCheckpoint(
-            actor_id.data, checkpoint_id))
+        cdef CActorID c_actor_id = actor_id.data
```

Comment:
Btw, I don't think it is necessary to do this is it? Because `ActorID` should already be a C++ object.

Reply:
I'm not sure about this. I thought `ActorID` was a Python object.